test/validation_test.ts

```diff
@@ -85,7 +85,7 @@ describe('Property validation in objects', () => {
     examples: [],
     parameters: [makeBooleanParameter('boolParam', 'Pass in a boolean (malformed is ok)')],
     execute: async ([boolParam]) => {
-      return {bool: boolParam};
+      return {Bool: boolParam};
```

Comment:
Well, the schema is also being normalized, so `bool: {type: ValueType.Bool}` is actually `Bool: {type: ValueType.Bool}` in the tests...

I can revert these changes if you'd like, but thought it was easier to keep them around since it's going to have to be changed anyway.

Reply:
As discussed, I reverted most of the test changes here and will let someone clean it up once we remove this flag. 

I retained the VM ones, since I didn't think it was necessary to add the normalization logic since these didn't appear in the user-visible test framework(?).